backend/src/Edna/Config/Definition.hs

```diff
@@ -35,6 +37,8 @@ data ApiConfig = ApiConfig
 data DbConfig = DbConfig
   { _dbConnString :: ConnString
   , _dbMaxConnection :: Int
+  , _dbInitDb :: DatabaseInitOption
+  , _dbInitScript :: FilePath
```

Comment:
Oh, more more thing. I think it should actually be optional because if `init-db` is set to `"disable"` this file is ignored. So I think the UX will be better if the person who doesn't want to init db doesn't need to think whether they need to put something sane into `init-script`.
That said, I think our whole configuration approach is quite unstable and imperfect, we will refine it after MVP (or even during MVP development if it appears to cause too many troubles), so you can ignore this comment for now.
I think it makes sense to do it if it takes ≈5 minutes. If it takes more, it's not worth the effort.